install.sh

```
@@ -263,6 +264,60 @@ setup_cursor() {
+  touch "$codex_config"
+
+  if grep -q '\[mcp_servers\.mnemo\]' "$codex_config" 2>/dev/null; then
+    ok "~/.codex/config.toml: mnemo MCP already configured"
+  else
+    tail -c1 "$codex_config" | grep -q $'\n' || printf '\n' >> "$codex_config"
+    printf '\n[mcp_servers.mnemo]\ncommand = "%s"\nargs = ["mcp", "--tools=agent"]\n' "$mnemo_bin" >> "$codex_config"
+    ok "~/.codex/config.toml: mnemo MCP configured"
+  fi
+
+  if grep -q 'codex_hooks' "$codex_config" 2>/dev/null; then
+    ok "~/.codex/config.toml: codex_hooks already set"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)